.gitmodules

```diff
@@ -34,3 +34,6 @@
 [submodule "libs/dxflib"]
     path = libs/dxflib
     url = https://github.com/LibrePCB/dxflib.git
+[submodule "libs/corrosion"]
+    path = libs/corrosion
+    url = https://github.com/LibrePCB/corrosion.git
```

Comment:
Why via submodule and not just install it in the Docker containers? I guess because of the old Ubuntu versions?

Reply:
If you consider any Ubuntu version older than 24.10 as *old*, then yes. Not even the latest LTS 24.04 has corrosion in the package management as far as I can see.

And also simply because this way it's under our control which exact version of corrosion is used, and if needed we have the ability to include our own patches.

Reply:
Ah and I didn't even talk about Windows. I'd expect having corrosion part of our sources is 100 times easier than installing it system-wide on a Windows...

Reply:
I guess this won't be a distribution problem, since it is only required at build time?

Reply:
Probably yes, but it should also be easy to *compile* LibrePCB on Windows. I do need this by myself when debugging Windows problems. I guess there isn't any real downside of having corrosion added as a submodule (the clone time for this repo shouldn't be an issue) but it saves a lot of troubles on systems where corrosion is not easily available.